src/main/java/com/saucelabs/bamboo/sod/action/PostBuildAction.java

```diff
@@ -37,6 +36,8 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import static com.saucelabs.bamboo.sod.config.SODKeys.*;
```

Comment:
Can you adjust your editor to not do wildcard imports

https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports